common-model/src/main/java/com/sequenceiq/common/api/type/LoadBalancerType.java

```diff
@@ -1,6 +1,6 @@
 package com.sequenceiq.common.api.type;
 
 public enum LoadBalancerType {
-    PUBLIC,
-    PRIVATE
+    ENDPOINT_ACCESS_GATEWAY,
```

Comment:
The same here, do we really need this refactor?
On the other hand for me PUBLIC/PRIVATE is more clear and more fit into CB's domain.

Reply:
This change has been reverted in https://github.com/hortonworks/cloudbreak/pull/9693.